v2/docs/gs/cli.md

```diff
@@ -123,16 +140,16 @@ hse storage add /var/bulk/kvdb1 storage.staging.path=/var/fast/staging1
 
 The staging media class directory `/var/fast/staging1` is configured for
 the KVDB home directory `/var/bulk/kvdb1`.
-
-This command can only be used to add a staging media class to a KVDB.
-The command will fail if either the KVDB already has a staging media class
-or if an application has the KVDB open.
-
 The next time an application opens the KVDB, the newly added
 staging media class will be used for KVS storage as determined by the
 [`mclass.policy`](params.md#kvs-runtime-parameters)
 parameter for each KVS.
 
+!!!warning
+    This command will fail if attempting to create an
+    [invalid storage configuration](storage.md#valid-configurations), or
+    if an application has the KVDB open.
+    
```

Comment:
For a KVDB configured with only a pmem media class, should we also say that a staging media class cannot be added without adding a capacity media class first?

Reply:
Adding a staging media class to a KVDB with only a pmem media class will fail because (if allowed) it would result in an invalid configuration.  I was trying to generalize the wording here to future-proof it for other media class additions or restrictions (though admittedly that doesn't  seem likely).  Maybe I'll add the pmem+staging as an example and keep the generalized language.

Reply:
Done, added example.